module/VuFind/src/VuFind/RecordDriver/DefaultRecord.php

```diff
@@ -1826,4 +1826,23 @@ public function getRecordDataFormatterSpecClass(): ?string
     {
         return \VuFind\RecordDataFormatter\Specs\DefaultRecord::class;
     }
+
+    /**
+     * Get a field as an array
+     *
+     * @param string $field Field
+     *
+     * @return array
+     */
+    protected function getFieldAsArray(string $field): array
+    {
+        // Make sure to return only non-empty values:
+        $value = $this->fields['description'] ?? '';
```

Comment:
I think there's a bit of missing refactoring here:

```suggestion
        $value = $this->fields[$field] ?? '';
```

Reply:
Oops! Fixed.